src/cli.js

```diff
@@ -86,6 +86,7 @@ commander
   .command("start")
   .alias("s")
   .description("start everything")
+  .option("-M, --entrypoints <entrypoints>", "Enter specific")
```

Comment:
Should we remove this line now that it was added above with `entrypointsOption`?